src/album/album.controller.ts

```diff
@@ -0,0 +1,36 @@
+import { Controller, Get, Post, Body, Put, Param, Delete, HttpCode } from '@nestjs/common';
+import { AlbumService } from './album.service';
+import { CreateAlbumDto } from './dto/create-album.dto';
+import { UpdateAlbumDto } from './dto/update-album.dto';
+
+@Controller('album')
+export class AlbumController {
+  constructor(private readonly albumService: AlbumService) {}
+
+  @Post()
+  @HttpCode(201)
```

Comment:
Furthermore, the response's status code is always 200 by default, except for POST requests which use 201.